models/fv_rs_cons_if.go

```diff
@@ -0,0 +1,99 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+)
+
+const (
+	DnfvRsConsIf        = "uni/tn-%s/ap-%s/epg-%s/rsconsIf-%s"
+	RnfvRsConsIf        = "rsconsIf-%s"
+	ParentDnfvRsConsIf  = "uni/tn-%s/ap-%s/epg-%s"
+	FvrsconsifClassName = "fvRsConsIf"
+)
+
+type ContractInterface struct {
+	BaseAttributes
+	NameAliasAttribute
```

Comment:
Object does not support nameAlias so we should remove the NameAliasAttribute from here and everywhere.